newIDE/app/src/Profile/Subscription/PlanCard.js

```diff
@@ -14,59 +18,197 @@ import { useResponsiveWindowWidth } from '../../UI/Reponsive/ResponsiveWindowMea
+              {price.periodCount} weeks
+            </Trans>
+          </Text>
+        );
+      } else {
+        return (
+          <Text key="week" noMargin color="secondary">
+            <Trans>
+              {price.amountInCents / 100}
+              {price.currency === 'EUR' ? '€' : '$'} every {price.periodCount}{' '}
+              weeks
+            </Trans>
+          </Text>
+        );
+      }
+    }
+  } else if (price.period === 'month') {
+    if (price.periodCount === 1) {
+      if (price.isPerUser) {
+        return (
+          <Text key="month" noMargin color="secondary">
+            <Trans>
+              {price.amountInCents / 100}
+              {price.currency === 'EUR' ? '€' : '$'}/seat/month
+            </Trans>
+          </Text>
+        );
+      } else {
+        return (
+          <Text key="month" noMargin color="secondary">
+            <Trans>
+              {price.amountInCents / 100}
+              {price.currency === 'EUR' ? '€' : '$'}/month
+            </Trans>
+          </Text>
+        );
+      }
+    } else {
+      if (price.isPerUser) {
+        return (
+          <Text key="month" noMargin color="secondary">
+            <Trans>
+              {price.amountInCents / 100}
+              {price.currency === 'EUR' ? '€' : '$'}/seat every{' '}
+              {price.periodCount} months
+            </Trans>
+          </Text>
+        );
+      } else {
+        return (
+          <Text key="month" noMargin color="secondary">
+            <Trans>
+              {price.amountInCents / 100}
+              {price.currency === 'EUR' ? '€' : '$'} every {price.periodCount}{' '}
+              months
+            </Trans>
+          </Text>
+        );
+      }
+    }
+  } else {
+    if (price.periodCount === 1) {
+      if (price.isPerUser) {
+        return (
+          <Text key="year" noMargin color="secondary">
+            <Trans>
+              {price.amountInCents / 100}
+              {price.currency === 'EUR' ? '€' : '$'}/seat/year
+            </Trans>
+          </Text>
+        );
+      } else {
+        return (
+          <Text key="year" noMargin color="secondary">
+            <Trans>
+              {price.amountInCents / 100}
+              {price.currency === 'EUR' ? '€' : '$'}/year
+            </Trans>
+          </Text>
+        );
+      }
+    } else {
+      if (price.isPerUser) {
+        return (
+          <Text key="year" noMargin color="secondary">
+            <Trans>
+              {price.amountInCents / 100}
+              {price.currency === 'EUR' ? '€' : '$'}/seat every{' '}
+              {price.periodCount} years
+            </Trans>
+          </Text>
+        );
+      } else {
+        return (
+          <Text key="year" noMargin color="secondary">
+            <Trans>
+              {price.amountInCents / 100}
+              {price.currency === 'EUR' ? '€' : '$'} every {price.periodCount}{' '}
```

Comment:
Probably worth a `formatCurrency(price.currency)` that returns €/$ and otherwise the currency as is.